docs/RELEASE.md

```diff
@@ -0,0 +1,154 @@
+# Release
+
+The outputs of a release are the `polkadot` and `polkadot-parachain` node binaries, the runtimes for Westend & Rococo and their system parachains, and new crate versions published to `crates.io`.
+
+# Setup
+
+We have two branches: `master` and `stable`. `master` is the main development branch where normal merge requests are opened. Developers need to mostly only care about this branch.  
```

Comment:
Non-breaking changes still can be added to stable on a more frequent cadence.

Fellowship could also if they wish use the latest audited nightly version if they want to incorporate breaking changes.

Reply:
>Fellowship could also if they wish use the latest audited nightly version if they want to incorporate breaking changes before the next stable release.

Yea true, did not think about this but the new process is meant to not slow us down. I guess audited nightly could work.

Reply:
I don't see how this could push any feature further back than 3 months and only then in the very worst case. Given our RC development schedules tend to be of the order of years it doesn't seem all that relevant.